graphql/schemas/Ecosystem/company.graphql

```diff
@@ -21,6 +21,7 @@ type Company {
     country_code: String
     created_at: DateTime
     updated_at: DateTime
+    count_agents: Int!
```

Comment:
@FredPeal why dont you had this the the guild dashboard not inside the compay?

Reply:
Mmm, this is a requirement for frontend.

Reply:
I thought that Jerly told with you. I Prefer to make another query for that using Lighthouse function agregate